examples/llama2/llama2_lmeval_onnx.json

```diff
@@ -0,0 +1,26 @@
+{
+    "input_model": {
+        "type": "HfModel",
+        "generative": true,
+        "model_path": "meta-llama/Llama-2-7b-hf",
+        "load_kwargs": { "attn_implementation": "eager" }
+    },
+    "systems": {
+        "local_system": {
+            "type": "LocalSystem",
+            "accelerators": [ { "device": "gpu", "execution_providers": [ "CUDAExecutionProvider" ] } ]
+        }
+    },
+    "passes": {
+        "mb": { "type": "ModelBuilder", "precision": "int4", "search": { "max_length": 2048, "min_length": 0 } }
+    },
+    "evaluators": {
+        "evaluator": { "type": "LMEvaluator", "tasks": [ "hellaswag" ], "batch_size": 1, "limit": 4, "max_length": 128 }
```

Comment:
I can see this evaluator is exactly same as `llama2_lmeval.json` evaluator. `evaluate_input_model` is true by default for this config, so this config will evaluate both pytorch model and onnx model?

Reply:
Yes! The branching happens within the evaluate function.